install_pim/manual/system_requirements/system_requirements.rst.inc

```diff
@@ -141,28 +141,32 @@ Elasticsearch is bundled with its own JDK version, but if you want to use the `n
 
 The following ports should be opened on the server host for the PIM to work properly:
 
-+---------------+-----------------------------------------------------------+---------------------------+
-| HTTP or HTTPS | 80 or 443                                                 | Required                  |
-+---------------+-----------------------------------------------------------+---------------------------+
-| MySQL         | unix socket or 3306 (if MySQL server on a different host) | Required                  |
-+---------------+-----------------------------------------------------------+---------------------------+
-| Elasticsearch | 9200                                                      | Required                  |
-+---------------+-----------------------------------------------------------+---------------------------+
-| SSH           | 22                                                        | Required (for deployment) |
-+---------------+-----------------------------------------------------------+---------------------------+
++------------------+-----------------------------------------------------------+---------------------------+
```

Comment:
I would remove the whole section as it's definitely a security issue and not in the responsibility of the PIM.